website/docs/intersection-types.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: intersection-types
-title: Intersection Types
+title: T.all
```

Comment:
```suggestion
title: Intersection Types
sidebar_label: Intersection Types (T.all)
```

The equivalent change for "Union Types" keeps the current title, but changes the sidebar label.  I don't have a strong opinion here, but symmetry between the two seems like a good idea.